### DISCUSSION TITLE
google play Refunded purchase

### DISCUSSION THREAD

**Discussion by mohammadfard1365** [airsdk/Adobe-Runtime-Support#1851] [General]
A few days ago, a successful purchase was registered for my app
Unfortunately, after 3 days, Google Refunded the amount. What is the reason ??
Does it go back to implementing in-app payments????

![fff](https://user-images.githubusercontent.com/32836801/163712435-5b360b88-35e6-4288-8aee-738a9a8ff1eb.jpg)

@marchbold 
@ajwfrost

**Comment by marchbold** [comment]
Hi, Sounds like you aren't calling `finishPurchase` for the purchase. You need to do this now with play store transactions in order to "acknowledge" the purchase. If you don't do this the purchase will be automatically refunded in ~3days.
